packages/api/src/external/hie/reset-doc-query-progress.ts

```diff
@@ -68,3 +68,21 @@ export async function resetDocQueryProgress({
     });
   });
 }
+
+export function buildInterrupt({
+  patientId,
+  cxId,
+  log,
+}: {
+  patientId: string;
+  cxId: string;
+  log: typeof console.log;
+}) {
+  return async (reason: string): Promise<void> => {
+    log(reason + ", skipping DQ");
+    await resetDocQueryProgress({
+      patient: { id: patientId, cxId },
+      source: MedicalDataSource.CAREQUALITY,
```

Comment:
This is now being called by CQ and CW, we need to make this dynamic.

I suggest creating a shred function on each HIE's shared folder, which would call this one with the respective MedicalDataSource, so we don't need to set this on each call.